weltyt.py

```diff
@@ -44,6 +44,33 @@ def buy_plants(plants):
 
         print("Invalid input, write integer only please. ")
 
+def buy_pesticides(pesticides):
+    amount = input("How many pesticides do you want to buy? Write only integer please. ")
+
+    try:
+        amount = int(amount)
+
+        money_amount = cur.execute(
+            'SELECT amount  FROM resources WHERE type = "Money";'
+        ).fetchone()[0]
+        pesticides_price = cur.execute(
+            'SELECT price FROM resources WHERE type = "Pesticides";'
+        ).fetchone()[0]
+        new_money_amount = money_amount - amount * pesticides_price
+        cur.execute(
+            f'UPDATE resources SET amount = {new_money_amount}  WHERE type = "Money";'
+        )
+        pesticides_amount = cur.execute(
```

Comment:
I would consider creating a generic function for this whole update money/resource logic, as DRY as possible.